js/core/dom_component.js

```diff
@@ -191,21 +191,21 @@ const DOMComponent = Component.inherit({
     },
 
     _attachDimensionChangeHandlers() {
-        const $el = this.$element();
+        const $element = this.$element();
```

Comment:
I use this short name :)

Reply:
In this component other methods use name `$element`. Should we change all `$element` to `$el`?

Reply:
I think I'll rollback these changes and will refactor this component in other PR